client/java/src/main/java/com/linecorp/centraldogma/client/Watcher.java

```diff
@@ -24,9 +24,14 @@
 import java.util.concurrent.TimeoutException;
 import java.util.function.BiConsumer;
 import java.util.function.Consumer;
+import java.util.function.Function;
 
+import javax.annotation.Nonnull;
```

Comment:
You don't need to use this annotation because everything is non-null by default here.

Reply:
👍